### PR TITLE
jasminejs normalcase test

### PR DESCRIPTION
**Pull request by ocampd1**
Added one test to normalcase-method.

### CODE REVIEW

**Comment by agamble33** [comment]
Check test run by Dave it checked out and was a positive result

**Review by agamble33 — DISMISSED** [review]
Please edit and make sure there are no spaces between variables

File: src/test/javascript/stringExtensionSpec.js [hunk 1]
```diff
@@ -56,6 +56,14 @@ define(['bytepushers'], function(BytePushers) {
                 // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
                 expect(result).not.toBe("Hello bytePushers");
             });
+            it('can format a sentence into camel case with a single letter word', function () {
+                var result = "Hello captainfalcon".toCamelCase().toNormalCase();
+                // Would think it would do this
+                // Hello this is a sentence => helloThisIsASentence => Hello This Is A Sentence
+                // Actually does this
+                // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
```
Comment (agamble33) [inline src/test/javascript/stringExtensionSpec.js]: placement of comments could be minimized to one line if possible and no space between var and expect.
```suggestion
                // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
```

**Review by agamble33 — CHANGES_REQUESTED** [review]
placement of comments could be minimized to one line if possible and no space between var and expect.

File: src/test/javascript/stringExtensionSpec.js [hunk 2]
```diff
@@ -56,6 +56,14 @@ define(['bytepushers'], function(BytePushers) {
...
+                expect(result).not.toBe("Hello captainFalcon");
```
Comment (agamble33) [inline src/test/javascript/stringExtensionSpec.js]: remove space from var and expect

**Review by agamble33 — CHANGES_REQUESTED** [review]
remove space from var and expect

File: src/test/javascript/stringExtensionSpec.js (hunk 1 above)
Comment (ocampd1) [reply src/test/javascript/stringExtensionSpec.js]: Left comments on initial it test and removed comments on following it tests to clean up code.

File: src/test/javascript/stringExtensionSpec.js [hunk 3]
```diff
@@ -1,63 +1,76 @@
 define(['bytepushers'], function(BytePushers) {
-    describe("String extension tests:", function() {
+    describe("String extension tests:", function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:2]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 4]
```diff
@@ -1,63 +1,76 @@
...
 
         var testStr = "Hello this is a random string";
 
         describe('String.prototype.trim', function () {
-            it('will trim whitespace correctly', function() {
+            it('will trim whitespace correctly', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:6]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 5]
```diff
@@ -1,63 +1,76 @@
...
                 var result = "                       l     shdasdk;lg;al j          ".trim();
 
                 expect(result).toBe("l     shdasdk;lg;al j");
             });
         });
 
         describe('String.prototype.includes', function () {
-            it('can tell if a string includes a string', function() {
+            it('can tell if a string includes a string', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:14]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 6]
```diff
@@ -1,63 +1,76 @@
...
                 var result = testStr.includes("o this i");
 
                 expect(result).toBe(true);
             });
 
-            it('can tell if a string does not include a string', function() {
+            it('can tell if a string does not include a string', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:20]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 7]
```diff
@@ -1,63 +1,76 @@
...
                 var result = testStr.includes("This is not in the string");
 
                 expect(result).toBe(false);
             });
         });
 
         describe('String.prototype.toCamelCase', function () {
-            it('can format a sentence into camel case', function() {
+            it('can format a sentence into camel case', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 8]
```diff
@@ -1,63 +1,76 @@
...
                 var result = "Hello this is a sentence.".toCamelCase();
 
                 expect(result).toBe("helloThisIsASentence.");
             });
         });
 
         describe('String.prototype.toNormalCase', function () {
-            it('can format a sentence into camel case', function() {
+            it('can format a sentence into camel case', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:28]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 9]
```diff
@@ -1,63 +1,76 @@
...
                 var result = "Hello world".toCamelCase().toNormalCase();
                 // helloWorld => "Hello World"
                 expect(result).toBe("Hello World");
             });
 
-            it('can format a sentence into camel case with a single letter word', function() {
-                var result = "Hello this is a sentence".toCamelCase().toNormalCase();
+            it('can format a sentence into camel case with a single letter word', function () {git
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js]: Remove space after function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 10]
```diff
@@ -1,63 +1,76 @@
...
+                var result = "Hello bytepushers".toCamelCase().toNormalCase();
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js]: Use the stringUndertTest, expectedResult, and actualResult variable naming/programming convention here.

File: src/test/javascript/stringExtensionSpec.js [hunk 11]
```diff
@@ -1,63 +1,76 @@
...
                 // Would think it would do this
                 // Hello this is a sentence => helloThisIsASentence => Hello This Is A Sentence
                 // Actually does this
                 // Hello this is a sentence => helloThisIsASentence => Hello This Is ASentence
-                expect(result).toBe("Hello This Is A Sentence");
+                expect(result).toBe("Hello Bytepushers");
+            });
+
+            it('can format a sentence into camel case with a single letter word', function () {
+                var result = "Hello bytepushers".toCamelCase().toNormalCase();
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js]: Use the stringUndertTest, expectedResult, and actualResult variable naming/programming convention here.

File: src/test/javascript/stringExtensionSpec.js [hunk 12]
```diff
@@ -1,63 +1,76 @@
...
+
+                expect(result).not.toBe("Hello bytePushers");
             });
-        });
 
-        describe('String.format', function () {
-            it('can format a string properly', function() {
-                var result = String.format("Hello {1} is {2} random {3}", "this", "a", "string");
+            it('can format a sentence into camel case with a single letter word', function () {
+                var result = "Hello captainfalcon".toCamelCase().toNormalCase();
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js]: Use the stringUndertTest, expectedResult, and actualResult variable naming/programming convention here.

File: src/test/javascript/stringExtensionSpec.js [hunk 13]
```diff
@@ -1,63 +1,76 @@
...
 
-                expect(result).toBe(testStr); // testStr = "Hello this is a random string";
+                expect(result).not.toBe("Hello captainFalcon");
+            });
+
+            describe('String.format', function () {
+                it('can format a string properly', function () {
+                    var result = String.format("Hello {1} is {2} random {3}", "this", "a", "string");
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:60]: Use the stringUndertTest, expectedResult, and actualResult variable naming/programming convention here.

File: src/test/javascript/stringExtensionSpec.js [hunk 14]
```diff
@@ -1,63 +1,76 @@
...
-                var result = String.format("Hello {1} is {2} random {3}", "this", "a", "string");
+            it('can format a sentence into camel case with a single letter word', function () {
+                var result = "Hello captainfalcon".toCamelCase().toNormalCase();
 
-                expect(result).toBe(testStr); // testStr = "Hello this is a random string";
+                expect(result).not.toBe("Hello captainFalcon");
+            });
+
+            describe('String.format', function () {
+                it('can format a string properly', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js:59]: Remove space between function and open parentheses.

File: src/test/javascript/stringExtensionSpec.js [hunk 15]
```diff
@@ -1,63 +1,76 @@
...
+                var result = "Hello bytepushers".toCamelCase().toNormalCase();
+
+                expect(result).not.toBe("Hello bytePushers");
             });
-        });
 
-        describe('String.format', function () {
-            it('can format a string properly', function() {
-                var result = String.format("Hello {1} is {2} random {3}", "this", "a", "string");
+            it('can format a sentence into camel case with a single letter word', function () {
```
Comment (tonte-pouncil) [inline src/test/javascript/stringExtensionSpec.js]: Remove space between function and open parentheses.

**Review by tonte-pouncil — CHANGES_REQUESTED** [review]

File: src/test/javascript/stringExtensionSpec.js (hunk 9 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 7 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 3 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:2]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 4 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:6]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 5 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:14]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 6 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:20]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 13 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:60]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 14 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:59]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 12 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 15 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 11 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 10 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 7 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js]: done

File: src/test/javascript/stringExtensionSpec.js (hunk 13 above)
Comment (agamble33) [reply src/test/javascript/stringExtensionSpec.js:60]: done

**Pull request merged**